internal/scm/utils/slice.go

```diff
@@ -0,0 +1,16 @@
+package utils
+
+func SliceContainsEntryFromSlice(slice1 []string, slice2 []string) bool {
+	slice1Map := make(map[string]*string, len(slice1))
```

Comment:
When creating a Set in Go, the idiomatic way is to use a 0 byte (empty) struct (`map[string]struct{}`)

Reply:
true, not sure what I was thinking there^^